serverless.yml

```diff
@@ -64,7 +64,7 @@ package:
 provider:
   name: aws
   runtime: nodejs12.x
-  memorySize: 2048
+  memorySize: 1024
```

Comment:
Are we sure this won't lead to API increased response times?

Reply:
reverted, might as well just to leave it